airbrakes/state.py

```diff
@@ -144,6 +144,9 @@ def update(self):
         # if our prediction is overshooting our target altitude, extend the airbrakes
         if self.context.apogee_predictor.apogee > TARGET_ALTITUDE:
             self.context.extend_airbrakes()
+        else:
+            # If our prediction is not overshooting, retract the airbrakes
+            self.context.retract_airbrakes()
```

Comment:
we need code here to not keep sending the command to the servo, like we had for interest launch. I don't know if that's a concern with the way the new servo is setup or if it's fine now and that wouldn't matter

Reply:
Generally with robotics/motor control, it is fine to keep sending the command. We do have the shitty threading stuff which in that case we don't want to, but for this launch we are gonna get rid of the no buzz I think.

Reply:
alright, then I would like us to get rid of that threading stuff before launch in another PR.

Reply:
I'm just gonna do it in this pr